src/object/srv_obj_migrate.c

```diff
@@ -2198,6 +2198,11 @@ migrate_enum_unpack_cb(struct dc_obj_enum_unpack_io *io, void *data)
 	if (!daos_oclass_is_ec(&arg->oc_attr))
 		return migrate_one_create(arg, io);
 
+	if (daos_oclass_is_valid(daos_obj_id2class(io->ui_oid.id_pub))) {
+		D_WARN("Skip invalid "DF_UOID".\n", DP_UOID(io->ui_oid));
```

Comment:
how could it get invalid oclass? does it mean the enumerate/pack with bug?

Reply:
According to the log, the OID seems invalid in this case. Unfortunately, the log is not enough here. So I add few extra debug log in this patch.